### DISCUSSION TITLE
Is composable moderation just blocklists?

### DISCUSSION THREAD

**Discussion by Cererean** [bluesky-social/atproto#1375] [Clarifications and "Why?"]
As in, is it built on the existing functionality for blocking? I'm interested in learning how it's intended to work but information is a bit thin on the ground. Will it be essentially a blacklist of users and a subscriber list, that instructs the services that they are not to be permitted to interact with each other?

I'm also wondering how long it will be. Custom feeds were brought in before federation -- is subscribable moderation expected to be the same?

**Comment by snarfed** [comment]
https://blueskyweb.xyz/blog/4-13-2023-moderation

**Reply by Cererean** [reply]
This seems to be talking a lot more about curation than moderation? More about what users see than who can interact with them.
